documentation/news.md

```diff
@@ -0,0 +1,6 @@
+# News & updates
+
+## 22 Sep 2022
+### Documentation update
+
+We've added documentation on the [ChildAware API](apps/childaware.md). You can use it to scope communication with (or between) dynamically available child nodes easily.
```

Comment:
Move into changelog?

Reply:
While they will sometimes will overlap, this section serves a different purpose than the changelog.